fastanime/cli/interfaces/anilist_interfaces.py

```diff
@@ -507,6 +511,12 @@ def provider_anime_episode_servers_menu(
         "[bold magenta] Episode: [/]",
         current_episode_number,
     )
+
+    # update discord activity for user
+    if config.discord == True:
+        discord_proc = multiprocessing.Process(target=discord_updater,args=(provider_anime_title,current_episode_number))
```

Comment:
Will test when I get back home, but MP normally does better about letting a child process run on its own then MT does. At least in my own use cases. MP will juts spawn a new proc on a different core than the one fastanime is running on.

Reply:
Doesn't look like it can be changed to a thread right now. Unable to get it to successfully stop. In testing on my system, Process uses about 150 MB of ram more than doing it with a thread. I don't see it as that big of a difference. I can tinker with doing the thread think if you want, and if I have any luck in getting it to stop the thread in the future, I can open a new pull request. to change that.

Reply:
Yeah 150mb is just to large. Definitely threads, is the way to go. Don't try killing the thread directly instead. Call the pypresence stop method. And thread will exit on its own

Reply:
finally got the threading to work....